index.html

```diff
@@ -9,36 +9,82 @@
   <title>Book App</title>
 </head>
 <body>
-  <div class="container w-75">
+  <nav class="navbar navbar-expand-lg navbar-dark" id="awesome-nav">
+    <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarTogglerDemo02" aria-controls="navbarTogglerDemo02" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <a class="navbar-brand"  onclick="document.getElementById('list-page').style.display = 'block'; document.getElementById('book-form').style.display = 'block'; document.getElementById('contact-page').style.display = 'block';">Awesome books<span class="ml-3 text-primary">|</span> </a>
 
-    <h1 class="text-center mt-3">All awesome Books</h1>
-    <table class="book-table table mt-3 table-striped">
-      <thead>
-        <tr>
-          <th>Book Title</th>
-        <th>Book Author</th>
-        <th></th>
-        </tr>
-      </thead>
-      <tbody id="book-list" class="mb-3"></tbody>
-    </table>
-    <h2 class="text-center mt-3">Add a new Book!</h2>
+    <div class="collapse navbar-collapse" id="navbarTogglerDemo02">
+      <ul class="navbar-nav mr-auto mt-2 mt-md-0 ">
+        <li class="nav-item">
+          <a class="nav-link"  onclick="document.getElementById('list-page').style.display = 'block'; document.getElementById('book-form').style.display = 'none'; document.getElementById('contact-page').style.display = 'none';">List <span class="ml-3 text-primary">|</span> </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" onclick="document.getElementById('list-page').style.display = 'none'; document.getElementById('book-form').style.display = 'block'; document.getElementById('contact-page').style.display = 'none';">Add new <span class="ml-3 text-primary">|</span></a>
+        </li>
+        <li class="nav-item" >
+          <a class="nav-link" onclick="document.getElementById('list-page').style.display = 'none'; document.getElementById('book-form').style.display = 'none'; document.getElementById('contact-page').style.display = 'block';">Contact</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+  <div id="page-time"></div>
+  <div class="container w-75">
+    <div id="list-page">
+      <h3 class="text-center mt-3">All awesome Books</h3>
+        <table class="book-table table mt-3 table-striped">
+          <thead>
+            <tr>
+              <th>Book Title</th>
+            <th>Book Author</th>
+            <th></th>
+            </tr>
+          </thead>
+          <tbody id="book-list" class="mb-3"></tbody>
+        </table>
+    </div>
+    
+    <form id="book-form">
+      <h4 class="text-center mt-3">Add a new Book!</h4>
 
     <div class="align">
-    <form id="book-form">
       <div class="form-group">
```

Comment:
- [ ] **[OPTIONAL]** Let's work on our HTML indentation. This is only one place where the indentation of nested HTML tags is unorganized. You may quickly fix your entire HTML document by formatting it with something like Visual Studio Code.